semantics.json

```diff
@@ -184,6 +184,14 @@
         "importance": "low",
         "default": true
       },
+      {
+        "name": "enableReset",
```

Comment:
If you change semantics.json, you'll have to take care of the language files, too.